scripts/households.py

```diff
@@ -74,7 +74,7 @@ def FOC_labor(c, n_s, args):
     Returns:
     foc_errors_n: A list of n2, n3, ..., nS
     '''
-    sigma, r, w, l_tilde, chi, b_ellipse, upsilon = args
+    sigma, w, l_tilde, chi, b_ellipse, upsilon = args
```

Comment:
@adibalaram you are absolutely right that `r` doesn't need to be in here.  Removing it is causing a test to fail.  To make this test pass, you need to go to `tests/test_households.py` and remove `r` from `n_args` on line 17.